base/reduce.jl

```diff
@@ -829,11 +829,11 @@ findmax(itr) = _findmax(itr, :)
 _findmax(a, ::Colon) = mapfoldl( ((k, v),) -> (v, k), _rf_findmax, pairs(a) )
```

Comment:
This could be `_findmax(a, ::Colon) = findmax(identity, a)`

And obviously below for `_findmin`.